.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,77 @@
+# Release vFIXME
+
+## Checklist
+
+- [ ] [Create a new github milestone](https://github.com/DataDog/dd-trace-go/milestones/new) for the next version and
+  move any open PRs and issues that are in the current release milestone to the new one.
+
+- [ ] Draft [the release note](https://github.com/DataDog/dd-trace-go/releases/new) and ask the teams for its review on
+  the `#guild-dd-go` channel.
+
+- Deploy a release candidate:
+    - Create a release candidate version on this release branch:
+        - [ ] Bump and commit the release candidate version string in `/internal/version/version.go` (eg. `v1.2.3-rc.4`)
+        - [ ] Tag the bump commit with the release candidate version (eg. `git tag v1.2.3-rc.4`)
+        - [ ] Push the tag and commit (eg. `git push --tags origin release-v1.2.3`)
+    - Deploy some Datadog services to staging:
+        - [ ] Give a heads-up on the channel `#staging-headsup` that you are starting the deployment.
+        - [ ] Create a draft pull request on dd-go upgrading dd-trace-go to the release candidate tag:
+          ```console
+          dd-go/$ git checkout prod
+          dd-go/$ git pull
+          dd-go/$ git checkout -b dd-trace-go-v1.2.3
+          dd-go/$ go get -u -v gopkg.in/DataDog/dd-trace-go.v1@v1.2.3-rc.4
+          dd-go/$ go mod tidy
+          dd-go/$ git add go.mod go.sum
+          dd-go/$ git commit -m '[go.mod] upgrade dd-trace-go to v1.2.3-rc.4'
+          dd-go/$ git push -u origin dd-trace-go-v1.2.3
+          ```
+        - [ ] Deploy some services to staging:
+          ```console
+          dd-go/$ env GITLAB_TOKEN=MY_TOKEN to-staging trace-sampler \
```

Comment:
Can you clarify what this first line is doing (with the GITLAB_TOKEN=MY_TOKEN)?

Reply:
Now clarified 👍